addOns/replacer/src/main/java/org/zaproxy/zap/extension/replacer/ReplacerParam.java

```diff
@@ -50,6 +50,29 @@ public class ReplacerParam extends AbstractParam {
 
     private static final String CONFIRM_REMOVE_RULE_KEY = REPLACER_BASE_KEY + ".confirmRemoveToken";
     private static final String FALSE_STRING = "false";
+    private static final String TRUE_STRING = "true";
+
+    protected static final String REPORT_TO_DESC = "Disable Report-To or Report-Uri (CSP, etc)";
+    protected static final String REPORT_TO_REGEX = "(?i)report-(?:to|uri)";
+    protected static final String REPORT_TO_REPLACEMENT = "report-disabled";
```

Comment:
Isn't this going to potentially cause validation issues? Why not properly remove the headers?

Reply:
Not that I experienced. Though I only surfed with it a bit. The browser should just ignore it since it won't match what's expected.

As for "properly" removing it do you mean matching from report-to|uri up till the next semi-colon and replacing with empty string? I guess we could do that, but then the full Report-To header would require a second rule.

Reply:
Fine, we can adjust if there are any issues later.